### ISSUE TITLE
Missing Button "Homematic IP Access Points"

### ISSUE THREAD

**Issue by helmsklamm** [jens-maus/RaspberryMatic#573]
**Describe the bug**
Homematic IP Access points  button missing in "Systemsteuerung" although I am running the current version: Momentan installierte Version: 3.43.15.20190223.
Maybe I am totally on the wrong track but from my understanding, this Firmware version should already support HM IP Wired.
I have no HM IP Access point but I thought it is not necessary  in case of the HM IP Wired version, becasue the DRAP is connected to my Home LAN via RJ45 Cable and my Router shows the device online.

**To Reproduce**
1. Go to System Settings
2. Search for Button "Homematic IP Access points" as descibed in the manual for HM IP Wired DRAP (Chapter 7.6.1 "Anlernen"). I don't see the button there. Something interesting: The Button layout indicates that one button is missing or "hidden". See screenshot: The yellow area is empty as if it was reserved for the button? 
![image](https://user-images.githubusercontent.com/36575745/54071052-0f2ab580-4268-11e9-9f52-83bc32fb2f2a.png)

**Expected behavior**
Button should exist as per ELV DRAP Manual "(Wired Access Point (DE)"

**Screenshots**

**System information (please complete the following information):**
 - Version: Momentan installierte Version: 3.43.15.20190223
 - Hardware is RaspberryPi3

**Additional context**
I have tried 

- different Browsers

- Raspberrymatic restarts

- uninstall "Zusatzsoftware"

- Clear Browser Cache

- Reinstall latest Firmware

Nothing helped so far.

I am not connected via Cloud Service (Homematic Connect) but have an entirely local installation.

**Comment by jens-maus** [comment]
This button and support for HmIP-Wired will only be available if you have a RPI-RF-MOD RF-Module installed on your RaspberryPi/Tinkerboard. HmIP-Wired is NOT supported with the old HM-MOD-RPI-PCB!

**Comment by helmsklamm** [comment]
Well my gateway is quite new. Purchased it one year ago at piotek shop:
https://www.piotek.de/PioTek-Desktop-Pi-RaspberryMatic-Zentrale-incl-16GB-SDCARD

**Comment by libertyx82** [comment]
This Central is using the old HM-MOD-RPI-PCB Module

**Comment by helmsklamm** [comment]
Okay many thanks for Super quick support. Will of course donate as your work is really awesome.

**Comment by helmsklamm** [comment]
Dear homematic community.
I talked to the owner of piotek smart home Shop where I purchased the raspberrymatic device only about one year ago. 
It turned out that the expensive device (more than 200 Euro) can't even be upgraded to the new rf module. It wont fit into the casing and will not only affect me but alsomany other Users.
I am surprised that I am the only one who ran into the problem so far. 



The owner said he will get in contact with eq3 to clarify why the heck they restrict hm ip wired (a device where the Gateway is connected via cable) to  a certain Version of a rf module that should not even be required. 

Will raspberrymatic find a "solution" or workaround  for this restriction or is there any technical Dependency that I am not aware of? I googled a lot and found a few articles that mention the fact that the new rf module is required but no real explanation why this is the case.

Even very experienced Power users (owner of technikkram.net blog) seem not to have an idea why eq3 did this :
https://technikkram.net/2019/03/homematic-ip-wired-unterstuetzung-nur-mit-funkmodul-rpi-rf-mod

Does anybody know more about the background? Is it theoretically possible that a raspberrymatic update "fixes" the issue (i know its not a bug). 

This does not even make sense from eq3 perspective as it makes hm ip wired quite unattractive for many users...... 

Sorry for bothering but I am quite sure that many users will run into the same issue and will appreciate a clarification.
I will of course also update this thread, once I get Feedback from the piotek Shop.
